src/Bicep.MSBuild.E2eTests/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+    "compilerOptions": {
```

Comment:
>compilerOptions [](start = 5, length = 15)

I mostly set up the e2e project by copy/pasting existing things. @shenglol and @anthony-c-martin, I haven't done a whole lot of TS, so feel free to point out any weirdness.